core/arch/arm/include/mm/core_mmu.h

```diff
@@ -124,6 +124,7 @@
  * MEM_AREA_RES_VASPACE: Reserved virtual memory space
  * MEM_AREA_SHM_VASPACE: Virtual memory space for dynamic shared memory buffers
  * MEM_AREA_TA_VASPACE: TA va space, only used with phys_to_virt()
+ * MEM_AREA_DDR_OVERALL: Overall DDR address range
```

Comment:
Add comment  `candidate to dynamic shm.`

Reply:
OK, I will update it.